apps/tenant-management-webapp/src/app/pages/admin/services/script/addScriptModal.tsx

```diff
@@ -125,7 +125,7 @@ export const AddScriptModal = ({
           <GoAButton
             type="primary"
             testId="script-modal-save"
-            disabled={validators && validators.haveErrors()}
+            disabled={script.name === "" || validators && validators.haveErrors()}
```

Comment:
Can you enable Eslint plugin on your VS code.  I think your linting is not catching these minor things.  The double quote should be a single quote